tensornetwork/linalg/linalg_test.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+import time
+import pytest
+import jax.numpy as jnp
+import jax.config as config
+from tensornetwork.linalg import linalg
+from tensornetwork import backends
+from tensornetwork.backends.numpy import numpy_backend
+from tensornetwork.backends.jax import jax_backend
+#pylint: disable=no-member
+config.update("jax_enable_x64", True)
+
+np_real_dtypes = [np.float32, np.float16, np.float64]
+np_dtypes = np_real_dtypes + [np.complex64, np.complex128]
+np_int_dtypes = [np.int8, np.int16, np.int32, np.int64]
+np_uint_dtypes = [np.uint8, np.uint16, np.uint32, np.uint64]
+
+def test_eye():
+  """
+  Tests linalg.eye against np.eye.
+  """
+  N = 4
+  M = 6
+  name = "Jeffrey"
+  axis_names = ["Sam", "Blinkey"]
+  backend_list = ["jax", "numpy"]
+  for backend in backend_list:
+    for dtype in np_dtypes + np_int_dtypes + np_uint_dtypes:
+      tnI = linalg.eye(N, dtype=dtype, M=M, name=name, axis_names=axis_names,
+                       backend=backend)
+      npI = np.eye(N, M=M, dtype=dtype)
+      np.testing.assert_allclose(tnI.tensor, npI)
+
```

Comment:
also check if axis names are as expected

Reply:
and `name`

Reply:
and backend

Reply:
also in all other tests

Reply:
to test all backends you can use a @pytest.mark.parametrize(backend=['numpy','jax','tensorflow','pytorch']), or just use the backend fixture `backend`, i.e.

```python 
test_eye(backend): #the variable `backend` is defined by the pytest fixture
  ...

```

Reply:
In general we always test all backends (except `symmetric`)